packages/jokul/package.json

```diff
@@ -70,6 +72,10 @@
     "peerDependencies": {
         "react": "^18.0.0"
     },
+    "optionalDependencies": {
+        "@floating-ui/react": "^0.24.2",
+        "framer-motion": "^7.10.3"
+    },
```

Comment:
Er vi sikre på at vi kan ha disse som optional? Jeg har ikke noe umiddelbart bedre forslag, annet enn å bundle inn koden, men mener at best practice for optional dependencies er at koden skal håndtere at de ikke er der. I [dokumentasjonen til Node](https://docs.npmjs.com/cli/v10/configuring-npm/package-json#optionaldependencies) står det i hvert fall
> It is still your program's responsibility to handle the lack of the dependency.

Vet ikke hvor realistisk det er å gjøre dette for de komponentene det gjelder, da. Særlig `Tooltip` er fort litt tricky, og det er vel også den hvor innbaking av koden vil være tyngst.

Reply:
Tanken min var vel egentlig at vi ikke KAN håndtere at de ikke finnes. Default for pnpm (og så vidt jeg vet npm) er å installere dem i hvert fall men komponentene vil jo håndtere at de ikke finnes ved å throw'e 😛 

Jeg kan teste hva som skjer om vi bare har dem som straight up dependencies men jeg syns egentlig det også blir litt misvisende. Det beste hadde egentlig vært om vi klarte oss uten disse store avhengighetene.

Reply:
Ja, leste nå at de blir installert som default med `optionalDependencies`. I så fall tenker jeg at det er greit som det er 😊